### ISSUE TITLE
Add Test coverage matrix into ESDOC

### ISSUE THREAD

**Issue by priyankawadhwa** [esdoc/esdoc#277]
I am using ESDOC for documenting my JS scripts, can you help me with how to create badges like "build successful" and "test coverage" in my ESDOC. I have documented my test scripts as well.

Currently, I am getting the document coverage badge only under the source tab which ESDOC itself provides.

For reference : I wish to add a coverage badge similar to https://esdoc.org/esdoc/.

**Comment by typhonrt** [comment]
Use a service like [Codecov](https://codecov.io) and add the associated badge file which is created when you integrate coverage testing with whatever continuous integration service you are using to your `README.md` file. Codecov just published an official NPM module and instructions are located here:
https://github.com/codecov/example-node

I use Istanbul, Mocha, Travis CI, Codecov for running tests w/ coverage and uploading the results. 

Basically this is not an ESDoc specific issue, so it should be closed.

**Comment by h13i32maru** [comment]
A documentation coverage badge is put into root directory of generated output. You can use it.

Or register your repo to [ESDoc Hosting Service](https://doc.esdoc.org). And you can use `https://doc.esdoc.org/github.com/your-name/your-repo/badge.svg`
